packages/bundler-plugin-core/src/types.ts

```diff
@@ -97,6 +97,20 @@ export interface Options {
    * Example: `rollup-package`
    */
   bundleName?: string;
+
+  sentryOptions: {
+    /** Only send bundle stats to sentry (used within sentry webpack plugin). */
+    sentryOnly?: boolean;
```

Comment:
I think this should be called `enabled`, or `isEnabled`, and we call the object `sentry` instead of `sentryOptions`.

Reply:
I like the later idea, calling the object `sentry`. The issue with `isEnabled` is that we want a flag that only runs the sentry stuff in case they don't have codecov setup. Maybe `skipCodecov` would be a better flag name?

Reply:
I put `isEnabled` and `sentryOnly` for maximum flexibility. isEnabled should be set to true on release, while `sentryOnly` should be set to true when we want to skip codecov